tests/testStatsdClient.cpp

```diff
@@ -114,6 +114,11 @@ void testSendRecv(uint64_t batchSize, uint64_t sendInterval) {
         throwOnError(client);
         expected.emplace_back("sendRecv.titi:3|g");
 
+        // Record a gauge "titifloat" to 123.456
+        client.gauge("titifloat", 123.456);
+        throwOnError(client);
+        expected.emplace_back("sendRecv.titifloat:123.456|g");
```

Comment:
We should check that the precision is satisfied:
```suggestion
        // Record a gauge "titifloat" to 123.456789 with precision 3
        client.gauge("titifloat", 123.456789);
        throwOnError(client);
        expected.emplace_back("sendRecv.titifloat:123.457|g");
```

Reply:
Made this change manually.